rplugin/python3/molten/moltenbuffer.py

```diff
@@ -325,7 +325,8 @@ def update_interface(self) -> None:
 
         if self.options.virt_text_output:
             for span, output in self.outputs.items():
-                output.show_virtual_output(span.end)
+                offset = span.calculate_output_offset()
```

Comment:
Doing this with virtual text can cause some weird behavior, although it's not really the end of the world.

![image](https://github.com/benlubas/molten-nvim/assets/56943754/79262f93-bf11-407b-9d8b-c3564cc440b0)

Reply:
I think I figured it out.